megamek/src/megamek/client/ui/swing/StatusBarPhaseDisplay.java

```diff
@@ -350,11 +350,11 @@ public String getRemainingPlayerWithTurns() {
         if (r > 0) {
             String m = "";
             int gti = clientgui.getClient().getGame().getTurnIndex();
-            List<GameTurn> gtv = clientgui.getClient().getGame().getTurnVector();
+            List<GameTurn> gtv = clientgui.getClient().getGame().getTurnsList();
```

Comment:
could be changed to `gtl` to match the naming convention we've used elsewhere.

Reply:
This is completely rewritten in another branch that is waiting for PR